docs/hub/models-widgets.md

```diff
@@ -71,8 +71,40 @@ widget:
   example_title: Custom Speech Sample 1
 ```
 
+But even more convenient, if the file lives in the corresponding model repo, you can just use the filename or file path inside the repo:
+```yaml
+widget:
+- src: sample1.flac
+  example_title: Custom Speech Sample 1
+```
+
 We provide example inputs for some languages and most widget types in [the DefaultWidget.ts file](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/DefaultWidget.ts). If some examples are missing, we welcome PRs from the community to add them!
 
+## Example outputs
+
+As an extension to example inputs, for each widget example, you can also optionally describe the corresponding model output, direcly in the `output` property.
+
+This is useful when the model is not yet supported by the Inference API (for instance, the model library is not yet supported, or the model is too large), so that the model page can still showcase how the model works and what results it gives.
+
+For instance, for an automatic-speec-recognition model:
+
+```yaml
+widget:
+- src: sample1.flac
+  output: "Hello my name is Julien"
+```
+
+The `output` property supports:
+- a JSON-encoded string that represents the full Inference API output
```

Comment:
i think i'll drop this, i don't think it's that useful after all^

Reply:
Hmm, might still be useful no? For example a text classification model where the output is a simple JSON with a list of `label` + `score`?

Reply:
Or do you mean removing json encoding but still support YAML? That would do actually

Reply:
yes still support YAML – see https://github.com/huggingface/moon-landing/pull/7602 (internal)

Reply:
just pushed 611a610341f919ae8ec435f8f00a9d9e11aa1a5a